hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -11,12 +11,13 @@ import (
 
 	ignitionapi "github.com/coreos/ignition/v2/config/v3_1/types"
 	"github.com/go-logr/logr"
+	"github.com/google/uuid"
 	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
 	"github.com/openshift/hypershift/control-plane-operator/releaseinfo"
 	"github.com/openshift/hypershift/hypershift-operator/controllers/manifests"
-	"github.com/openshift/hypershift/hypershift-operator/controllers/manifests/controlplaneoperator"
 	"github.com/openshift/hypershift/hypershift-operator/controllers/manifests/ignitionserver"
 	hyperutil "github.com/openshift/hypershift/hypershift-operator/controllers/util"
+	ignservercontrollers "github.com/openshift/hypershift/ignition-server/controllers"
 	capiv1 "github.com/openshift/hypershift/thirdparty/clusterapi/api/v1alpha4"
```

Comment:
I suggest we avoid introducing this dependency. I'd rather duplicate the constant here than have this dependency or introduce a dependency between the proxy and the api (if we were to extract it there)

Reply:
done